salesforce/client.bal

```diff
@@ -69,6 +65,250 @@ public isolated client class Client {
+    isolated remote function getBasicInfo(@display {label: "sObject Name"} string sobjectName)
+                                                returns @display {label: "sObject Basic Information"}
+                                                SObjectBasicInfo|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sobjectName]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Completely describes the individual metadata at all levels for the specified object. Can be used to retrieve
+    # the fields, URLs, and child relationships.
+    #
+    # + sObjectName - sObject name value
+    # + return - `SObjectMetaData` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Description"}
+    isolated remote function describe(@display {label: "sObject Name"} string sObjectName)
+                                            returns @display {label: "sObject Metadata"} SObjectMetaData|error {
+
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, sObjectName, DESCRIBE]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Query for actions displayed in the UI, given a user, a context, device format, and a record ID.
+    #
+    # + return - `SObjectBasicInfo` record if successful or else `sfdc:Error`
+    @display {label: "Get sObject Platform Action"}
+    isolated remote function getPlatformAction() returns @display {label: "sObject Basic Information"}
+                                                SObjectBasicInfo|error {
+        string path = utils:prepareUrl([API_BASE_PATH, SOBJECTS, PLATFORM_ACTION]);
+        return check self.salesforceClient->get(path);
+    }
+
+    //Describe Organization
+    # Lists summary details about each REST API version available.
+    #
+    # + return - List of `Version` if successful. Else, the occured Error.
+    @display {label: "Get Available API Versions"}
+    isolated remote function getApiVersions() returns @display {label: "Versions"} Version[]|error {
+        string path = utils:prepareUrl([BASE_PATH]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Lists the resources available for the specified API version.
+    #
+    # + apiVersion - API version (v37)
+    # + return - `Resources` as map of strings if successful. Else, the occurred `Error`.
+    @display {label: "Get Resources by API Version"}
+    isolated remote function getResources(@display {label: "API Version"} string apiVersion)
+                                                    returns @display {label: "Resources"} map<string>|error {
+        string path = utils:prepareUrl([BASE_PATH, apiVersion]);
+        json res = check self.salesforceClient->get(path);
+        return toMapOfStrings(res);
+    }
+
+    # Lists the Limits information for your organization.
+    #
+    # + return - `OrganizationLimits` as map of `Limit` if successful. Else, the occurred `Error`.
```

Comment:
This should be error

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sfdc/commit/666dfedf90963e702c3e32712fa8e2e41f087f2e